portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,26 +32,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> messages = new ArrayList<>();
-  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    messages.add("Hello!");
-    messages.add("Hola!");
-    messages.add("Hallo!");
+    Query query = new Query("Comments").addSort("timestamp", SortDirection.ASCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+		PreparedQuery results = datastore.prepare(query);
+
+		ArrayList<String> commentsArrayList = new ArrayList<>();
 
-    String json = convertToJson(messages);
+		for (Entity entity: results.asIterable()) {
```

Comment:
nitpick: Standard style is a space on both sides of the colon, like "for (Foo f : results)"